srcpkgs/xmonad/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'xmonad'
+pkgname=xmonad
+version=0.13
+revision=1
+build_style=haskell-stack
+stackage=lts-11.17
+hostmakedepends="ghc stack"
+makedepends="libX11-devel libXinerama-devel libXrandr-devel"
+depends="libX11 libXinerama libXrandr"
```

Comment:
shouldn't these be detected by shlibs ?

Reply:
I don't quite get how they should be detected by shlibs with the manual, but anyway I couldn't build the package without them.

Reply:
```
=> xmonad-0.13_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libm.so.6 <-> glibc>=2.26_1
   SONAME: librt.so.1 <-> glibc>=2.26_1
   SONAME: libdl.so.2 <-> glibc>=2.26_1
   SONAME: libpthread.so.0 <-> glibc>=2.26_1
   SONAME: libXinerama.so.1 <-> libXinerama>=1.0.3_1
   SONAME: libX11.so.6 <-> libX11>=1.2_1
   SONAME: libXrandr.so.2 <-> libXrandr>=1.3.0_1
   SONAME: libgmp.so.10 <-> gmp>=5.0.1_1
   SONAME: libffi.so.6 <-> libffi>=3.1_1
   SONAME: libc.so.6 <-> glibc>=2.26_1
```

you don't need libraries that are called by the "binary" in the depends=

Reply:
so yeah, remove the depends= line we just need their development packages when building which is part of makedepends=

Reply:
okay, it's ready now I guess